### DISCUSSION TITLE
悦跑圈导出数据时无反应

### DISCUSSION THREAD

**Discussion by HuangZhangxin** [yihong0618/running_page#438] [Q&A]
进行到这行命令时python3 scripts/joyrun_sync.py 152xxxxxxxx xxxxxx --with-gpx
回车后无事发生
查看scripts中joyrun_sync.py，发现其中Line 88的https://api.thejoyrun.com网页提示{"ret":"102", "msg":"该APP版本过旧，请更新到最新版本。"}
不知道是否与这个有关？

**Comment by yihong0618** [comment]
@ben-29 我们看一下

**Reply by ben-29** [reply]
应该是在服务器加了版本验证。晚上看看

**Comment by ben-29** [comment]
@HuangZhangxin 试了下，限制好像取消了，我用最新的代码能正常登录，你重新操作试试

**Reply by HuangZhangxin** [reply]
可以了，谢谢。
